components/brave_vpn/pref_names.h

```diff
@@ -18,8 +18,12 @@ constexpr char kBraveVPNRegionList[] = "brave.brave_vpn.region_list";
 constexpr char kBraveVPNDeviceRegion[] = "brave.brave_vpn.device_region_name";
 constexpr char kBraveVPNSelectedRegion[] =
     "brave.brave_vpn.selected_region_name";
-constexpr char kBraveVPNShowDNSPolicyWarningDialog[] =
+#if BUILDFLAG(IS_WIN)
+constexpr char kBraveVpnShowDNSPolicyWarningDialog[] =
     "brave.brave_vpn.show_dns_policy_warning_dialog";
+#endif  // BUILDFLAG(IS_WIN)
+constexpr char kBraveVPNShowNotificationDialog[] =
```

Comment:
this one also belongs in IS_WIN

Reply:
done in https://github.com/brave/brave-core/pull/15989/commits/c2393ba383cbfdab6e199dff99215af777d2abfa